src/plugins/data/common/search/aggs/buckets/_interval_options.ts

```diff
@@ -20,12 +20,15 @@
 import { i18n } from '@kbn/i18n';
 import { IBucketAggConfig } from './bucket_agg_type';
 
+export const autoInterval = 'auto';
```

Comment:
Is there a reason why don't you use upper case for this variable?

Reply:
Sorry, but is there a reason why I cannot use that notation?  I don't see a clear rule

Reply:
I didn't find any rule in kibana style guide for this case 😄 
Does it mean we could use any naming format?
I wonder because I always do that in my code, and see others doing the same.

Reply:
I think you should  copy the style of the owners of this code. If we are talking about the data plugin they use this notation. In vis editor I see both